src/main/java/dev/emi/stepheightentityattribute/mixin/EntityAttributesMixin.java

```diff
@@ -0,0 +1,20 @@
+package dev.emi.stepheightentityattribute.mixin;
+
+import dev.emi.stepheightentityattribute.StepHeightEntityAttributeMain;
+import net.minecraft.entity.attribute.ClampedEntityAttribute;
+import net.minecraft.entity.attribute.EntityAttribute;
+import net.minecraft.entity.attribute.EntityAttributes;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+
+@Mixin(EntityAttributes.class)
+public abstract class EntityAttributesMixin {
```

Comment:
Why was this moved to a mixing, can you no longer register normally?

Reply:
Yep. It yelled at me for the registry being frozen while it was still in the mod initializer, so I moved it to the same place where the others are registered